components/prism-hcl.js

```diff
@@ -1,25 +1,26 @@
 Prism.languages.hcl = {
+	'comment': /(?:\/\/|#).*|\/\*[\s\S]*?(?:\*\/|$)/,
 	'heredoc': {
 		pattern: /<<-?(\w+)[\s\S]*?^\s*\1/m,
+		greedy: true,
 		alias: 'string'
 	},
-	'comment': /(?:\/\/|#).*|\/\*[\s\S]*?(?:\*\/|$)/,
 	'keyword': [
 		{
-			pattern: /(?:resource|data)\s+(?:[\w-]+|"[\w-]+")(?=\s+"[\w-]+"\s+{)/i,
+			pattern: /(?:resource|data)\s+(?:[\w-]+|"(?:\\[\s\S]|[^\\"])*")(?=\s+"[\w-]+"\s+{)/i,
```

Comment:
The lookahead still contains the old `"[\w-]+"`.

Reply:
Sorry, I missed them.